### DISCUSSION TITLE
Correct use of bale wrap/collect

### DISCUSSION THREAD

**Discussion by fudoido** [Courseplay/Courseplay_FS22#721] [Q&A]
Hello guys. Been trying this but can't figure out how to make it work, any help would be much appreciated.

I put the tractor with the Anderson round bale trailer in the field, either using the mini hud or the helper menu I set it to collect/wrap and start job, no other courses loaded (do I need to generate a course for it first?) and tractor keeps driving backwards in circles.

Which are the correct steps and what to avoid in options to make it work? Is is the function not yet implemented yet and being used as a placeholder at the moment?

Thanks in advance for any tips :)

**Comment by Tensuko** [comment]
Place it on a field and start it with wrap/collect is correct.
No course needed.
Do you use the latest version ?

**Comment by theRealPDXMike** [comment]
I'm confused... the Anderson WB-55 in FS22 is for baling POPLAR trees into wood chips. Is this what you are using to wrap grass to ferment into silage. Is there a modded version that does do grass?

**Reply by Tensuko** [reply]
That is baleing, not wrapping...

**Reply by theRealPDXMike** [reply]
oops I thought OP was trying to bail and wrap with the Anderson WB-55. Dang eyes, I see now he was using a trailer  to collect the bails

**Reply by fudoido** [reply]
yes it is the bale loading trailer for straw bales, not the wrapper.
i think i managed to get it working, for some reason it wasnt starting before. however i am noticing it stops quite a few times along the process, not sure if due the modded map (been playing on county line map), maybe fields too big for it to detect, or if its just the bales being too far apart from each other and CP stops for not detecting them nearby in the field.....

**Reply by fudoido** [reply]
and @Tensuko yes i always try to use the latest version. currently using .25
another heads-up, do not use more than one trailer in the same field, or at least dont make them work too close of each other. i got one trailer chasing the 2nd trailer trying to steal its bales lol. it was funny though XD

**Reply by Tensuko** [reply]
haha yea, but that will not work very well :)
